core/src/components/checkbox/checkbox.scss

```diff
@@ -248,6 +256,25 @@ input {
   max-width: 200px;
 }
 
+// Label Placement - Stacked
+// ----------------------------------------------------------------
+
+/**
+ * Label is on top of the checkbox.
+ */
+:host(.checkbox-label-placement-stacked) .checkbox-wrapper {
```

Comment:
The following applies to checkbox, radio, and toggle (range is not impacted).

When used in an item, there is not enough bottom margin. It looks like we used to account for this on `.label-text-wrapper` which works fine when the label and control are on the same line, but it looks a little smushed when using stacked layout:

![image](https://github.com/ionic-team/ionic-framework/assets/2721089/3a8db2ea-d7a0-4b25-879d-9815f11ddd79)

You'll want to apply margin on the top of `.label-text-wrapper` as well as margin on the bottom of `.native-wrapper`. So something like this:

```scss
:host(.in-item.toggle-label-placement-stacked) .label-text-wrapper {
  @include margin($toggle-item-label-margin-top, null, $form-control-label-margin, null);
}

:host(.in-item.toggle-label-placement-stacked) .native-wrapper {
  @include margin(null, null, $toggle-item-label-margin-bottom, null);
}
```

We'll want screenshot tests for this too.